pages/clusters/manage_clusters.md

```diff
@@ -77,6 +77,8 @@ You can set the _Allowed IP Addresses_ when creating a token, or you can modify
 
 Modifying the _Allowed IP Addresses_ forcefully disconnects any existing agents with IP addresses outside the updated value. This prevents the completion of any jobs in progress on those agents.
 
+This setting currently does not restrict access to the [Metrics API](/docs/apis/agent-api/metrics) for the given cluster token.
+
 There is a maximum of 24 CIDR blocks per agent token, and IPv6 is currently not supported.
```

Comment:
I generally try to avoid time words like "currently" in the docs because their meaning can be unclear and if something changes the false promise erodes trust. Perhaps we can combine these limitations with something like:

```suggestion
Note the following limitations:

- This setting does not restrict access to the [Metrics API](/docs/apis/agent-api/metrics) for the given cluster token.
- There is a maximum of 24 CIDR blocks per agent token, and IPv6 is currently not supported.
```

Reply:
Just separating out into three distinct points - will commit this, but lmk if you prefer you version @mbelton-buildkite 
```suggestion
Note the following limitations:

- This setting does not restrict access to the [Metrics API](/docs/apis/agent-api/metrics) for the given cluster token.
- There is a maximum of 24 CIDR blocks per agent token.
- IPv6 is currently not supported.
```